### DISCUSSION TITLE
Error: The query requires a COLLECTION_GROUP_ASC index for collection CRM_CONTACT and field twitter.

### DISCUSSION THREAD

**Discussion by lbjerga** [AntlerVC/firetable#800] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by harinij** [comment, marked as answer]
@lbjerga It appears that you are trying to sort by some field that is not yet indexed. If you click the URL suggested in message that pops up and follow it to create an index of the listed fields, then sorting on those fields will work. Let us know if you still continue to face this issue.
